Azkfile.js

```diff
@@ -31,4 +31,25 @@ systems({
       BUNDLE_APP_CONFIG: ".bundle",
     },
   },
+  mongodb: {
+    image : { docker: "azukiapp/mongodb" },
```

Comment:
it's not ruby, but javascript
it can be done with the following snippet, but I don't know if it's worth as is the only non ruby file we should have
```
javascript:
  enable: false
```